tests/test.cpp

```diff
@@ -157,8 +158,42 @@ flatbuffers::unique_ptr_t CreateFlatBufferTest(std::string &buffer) {
 void AccessFlatBufferTest(const uint8_t *flatbuf, size_t length) {
 
   // First, verify the buffers integrity (optional)
-  flatbuffers::Verifier verifier(flatbuf, length);
-  TEST_EQ(VerifyMonsterBuffer(verifier), true);
+  {
+    flatbuffers::Verifier verifier(flatbuf, length);
+    TEST_EQ(VerifyMonsterBuffer(verifier), true);
+  }
+
+  std::vector<uint8_t> test_buff;
+  test_buff.resize(length * 4 + 4);
+  std::memcpy(&test_buff[0], flatbuf , length);
+  std::memcpy(&test_buff[length], flatbuf , length);
+  std::memcpy(&test_buff[2 * length + 1], flatbuf , length);
```

Comment:
Remove the +1 +3 +4.. no need to test on un-aligned buffers, these aren't supported anyway.
Oh, and only 2 copies (rather than 4) are needed to test this.
Maybe use different verifier names rather than your use of {}